CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+Development version
```

Comment:
Alternatively, this could be 

```
Version 0.12 (in progress)
```

which more clearly signals that work listed here would be included in that release once published.

Reply:
I wasn't sure about this as it may actually be 0.11.1, no?

Reply:
True; ignore my comment.